codecept.conf.js

```diff
@@ -57,6 +57,7 @@ exports.config = {
     uploadStandardDirectionsDocumentEventPage: './e2e/pages/events/uploadStandardDirectionsDocumentEvent.page.js',
     sendCaseToGatekeeperEventPage: './e2e/pages/events/sendCaseToGatekeeperEvent.page.js',
     addHearingBookingDetailsEventPage: './e2e/pages/events/addHearingBookingDetailsEvent.page.js',
+    provideStatementOfServiceEventPage: './e2e/pages/events/provideStatementOfServiceEvent.page.js',
```

Comment:
addStatmentOfServiceEventPage* to match the event name

Reply:
👍